src/playground/Insights/EventsList/index.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { EventsList } from "."
+
+const meta: Meta = {
+  component: EventsList,
+  parameters: {
+    tags: ["autodocs"],
+  },
+  args: {
+    title: "Events",
+    events: [
+      {
+        title: "Product Review",
+        length: 2,
```

Comment:
List as a name might be too generic I think.

Reply:
Yes I agree. My point here is that maybe this component should be a InsightList or something similar, because the items can be other things than events. Like in Trainings or Status Insights.

![Screenshot 2024-08-29 at 10 00 00](https://github.com/user-attachments/assets/6e14ac07-c116-4a4a-a75b-7998178e4e08)
![Screenshot 2024-08-29 at 10 00 07](https://github.com/user-attachments/assets/d49c569b-a083-4d0e-a943-b3ba0b724dd7)

Reply:
Aren't those events too in a more generic meaning? Not Events as in "Factorial Events", but they are still events, right?

Reply:
Regarding the props, I changed it in the last commit.

Reply:
yes but why limit this list to events? I mean maybe in the future we want to show a list of tasks like this. But we can change it in the future if we need to show list another type

Reply:
I understand the concern that we might want to reuse it in the future for something that are not events. I can change the name perhaps to ItemsList which is more generic. However, I don't think the philosophy of F1 is to be generic, I think it's more to be specific and restrictive. Perhaps @sauldom102 or @eliseo-juan or even @lily-factorial can share their views on this.

Reply:
I think we should have it more generic as @arnaub suggests, if they are gonna look exactly the same and it will only change the title/subtitle structure we can make that distiction in the main repo components that we create tailored to each type of data like events, time-off, etc.

Reply:
Thank you both for your feedback, this component will probably change its name as we have more usecases and we find ourselves with more than one type of list. Next week we'll have a discussion and include @josepjaume about how to handle different types of lists, if they should have different names, if they should be the same component, etc.

On the meantime, I'm gonna merge it as it is just because I need to use some of the changes in the PR, but we'll revisit this in the next few days.